plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -403,12 +414,13 @@ func PodFitsResources(pod *api.Pod, nodeInfo *schedulercache.NodeInfo) (bool, er
 			newInsufficientResourceError(podCountResourceName, 1, int64(len(nodeInfo.Pods())), allowedPodNumber)
 	}
 	podRequest := getResourceRequest(pod)
-	if podRequest.milliCPU == 0 && podRequest.memory == 0 {
+	if podRequest.milliCPU == 0 && podRequest.memory == 0 && podRequest.nvidiaGPU == 0 {
```

Comment:
I don't see how `podRequest.nvidiaGPU == 0` and `podRequest.nvidiaGPU > 1` could both be true.  did you mean `||`?  If so, I don't think that is necessary.